docs/modules/ROOT/pages/how-to-guides/proc_prefetching-dependencies-to-support-hermetic-build.adoc

```diff
@@ -201,6 +203,39 @@ spec:
 
 include::../partials/con_hermetic_verification.adoc[]
 
+== Enabling prefetch builds for `npm`
+Cachi2 supports `npm` by fetching any dependencies you declare in your `package.json` and `package-lock.json` project files. The npm CLI manages the `package-lock.json` file automatically, and Cachi2 downloads the fetched dependencies to your `deps/npm/` output directory as ``.tar` files.
+
+.Prerequisites
+* You have an link:https://redhat-appstudio.github.io/docs.appstudio.io/Documentation/main/how-to-guides/configuring-builds/proc_upgrade_build_pipeline/[upgraded build pipeline].
+* You have an up-to-date link:https://docs.npmjs.com/cli/v9/configuring-npm/package-lock-json[`package-lock.json`] file in your source repository. To make sure that you have the latest `package-lock.json` file, or to create a lockfile, run the link:https://docs.npmjs.com/cli/v9/commands/npm-install?v=true[`npm-install`] command.
+* You have a `package.json` file that's newer than version 1.
+
+.Procedure
+To create a hermetic build for a component, complete the following steps:
+. Run the following command: 
+
++
+[source,command]
+----
+cachi2 fetch-deps \
+  --source ./my-repo \
+  --output ./cachi2-output \
+  '<_JSON input_>' <1>
+  ----
+<1> Here's an example of `JSON input` as shown in the previous `fetch-deps` command:
++
+[source,json]
+----
+{
+  // "npm" tells Cachi2 to process npm packages
+  "type": "npm",
+  // path to the package (relative to the --source directory)
+  // defaults to "."
+  "path": ".",
+}
```

Comment:
This is how one would use Cachi2 standalone, outside of Konflux. Within Konflux, all the user has to do is set the `prefetch-input` parameter, similar to the procedure for gomod and pip

Reply:
@chmeliik are these instructions concerned only with "within Konflux"?

Reply:
I think so - that's how the doc was conceived for gomod and pip as well